src/components/Modal/BaseModal.tsx

```diff
@@ -306,7 +306,7 @@ function BaseModal(
                             shouldPreventScroll={shouldPreventScrollOnFocus}
                         >
                             <View
-                                style={[styles.defaultModalContainer, modalPaddingStyles, modalContainerStyle, !isVisible && styles.pointerEventsNone]}
+                                style={[styles.defaultModalContainer, modalContainerStyle, modalPaddingStyles, !isVisible && styles.pointerEventsNone]}
```

Comment:
I need to switch this up because `modalContainerStyle` will override the padding style from `modalPaddingStyles`. `modalPaddingStyles` contains the `modalContainerStyle` padding + safe area.